homework-submission/my app/my-app/src/add.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+class AddTodo extends React.Component {
+  state = {
+    description: '',
+    done: false
+  }
+  todo = (element) => {
+    this.setState({ [element.target.id]: element.target.value })
```

Comment:
As the previous comment, this will be `element.target.name` which makes more sense for others.

And you will get more flexibility in your code.